test/e2e/volumes.go

```diff
@@ -839,6 +840,64 @@ var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
 	})
 
 	////////////////////////////////////////////////////////////////////////
+	// vSphere
+	////////////////////////////////////////////////////////////////////////
+
```

Comment:
What does this test do? Can you make "it" description more meaningful. Same in other places too.

Not happen with file name too.